src/scalaDebugger.ts

```diff
@@ -31,10 +31,13 @@ export async function start(
   noDebug: Boolean,
   ...parameters: any[]
 ): Promise<Boolean> {
-  return vscode.commands
-    .executeCommand<DebugSession>(
-      ServerCommands.DebugAdapterStart,
-      ...parameters
+  return commands
+    .executeCommand("workbench.action.files.save")
```

Comment:
I wonder if we should do this unconditionally, or prompt the user and offer to save.

Reply:
So it seems that vscode does it automatically anyways when the debug session is run, which means that even with prompting the user the end result will be the same.

Reply:
👍 alright then